src/targets/gpu/device/logsoftmax.cpp

```diff
@@ -11,9 +11,10 @@ inline namespace MIGRAPHX_INLINE_NS {
 namespace gpu {
 namespace device {
 
-void logsoftmax(hipStream_t stream, const argument& result, const argument& arg, int axis)
+void logsoftmax(hipStream_t stream, const argument& result, const argument& arg, int64_t axis)
```

Comment:
The axis should be computed in the operator and passed to `logsoftmax` device function.

Reply:
fixed accordingly